app/views/layouts/dashboard/_sidebar.html.erb

```diff
@@ -37,6 +37,11 @@
       </li>
 
       <% if current_user.staff_account && current_user.staff_account.has_role?(:admin, current_user.staff_account.organization) %>
+        <li class="nav-item">
+          <%= active_link_to edit_form_path, class: "nav-link" do %>
+            <i class="nav-icon fe fe-file-text me-2"></i> Application Form
```

Comment:
Is there a different icon we should use here?

Reply:
I think this one is fine - we could also use `file-plus`? I think we could also change the adopter/foster applications to `inbox`

https://feathericons.com/

Reply:
In case we don't settle on it we can use anything for now and make an issue later for icons once we've built the MVP out.